Kernel/CMakeLists.txt

```diff
@@ -389,22 +389,35 @@ if (NOT "${SERENITY_ARCH}" STREQUAL "aarch64")
         ${CRYPTO_SOURCES}
     )
 else()
+    set(PREKERNEL_SOURCES
+        Arch/aarch64/prekernel/PrekernelMMU.cpp
+        Arch/aarch64/prekernel/PrekernelExceptions.cpp
+        Arch/aarch64/prekernel/PrekernelCommon.cpp
+        Arch/aarch64/prekernel/boot.S
+        Arch/aarch64/prekernel/Aarch64_asm_utils.S
```

Comment:
These still need to be updated. Is your FS case insensitive by any chance? Otherwise this shouldn't even build...

Reply:
Yes it is! Lemme login to my Linux box and do this...